service/history/workflowExecutionContext.go

```diff
@@ -79,7 +79,9 @@ func (c *workflowExecutionContext) loadWorkflowExecution() (*mutableStateBuilder
 		Execution: c.workflowExecution,
 	})
 	if err != nil {
-		logging.LogPersistantStoreErrorEvent(c.logger, logging.TagValueStoreOperationGetWorkflowExecution, err, "")
+		if common.IsPersistenceTransientError(err) {
```

Comment:
just to confirm - do you want to log only on transient errors (or) only on errors other than transient errors ?

Reply:
Just the transient errors.  Any permanent errors like EntityNotExist should not be logged as errors.